### ISSUE TITLE
Multi-touch attribution

### ISSUE THREAD

**Issue by benjaminsavage** [patcg/ppa-api#44]
Advertisers may want the option to perform "multi-touch attribution", giving partial credit to multiple impressions which preceded a conversion. 

Retrieving all relevant impressions from the impression store is simple enough, and executing simple heuristics to allocate partial credit is simple enough. The return type would still be a histogram contribution, but potentially with non-zero values in multiple histogram bins (but the sum of all bins still being equal to the specified conversion value). 

Is this feature simple enough to make the cut for the Level 1 MVP product?

**Comment by martinthomson** [comment]
Given that we have to add a new Prio instantiation for this (#33), it looks easy enough to ensure that we can make multi-touch possible at that time.  We'd need to define the attribution logic, but I'd say that this is probably easy enough and valuable enough to make the cut.

**Comment by apasel422** [comment]
#218 added basic support for multi-touch. More advanced options for this are being discussed in #204.

**Comment by martinthomson** [comment]
I agree; let's mark this closed once we've had the discussion about what we've implemented so far.

**Comment by apasel422** [comment]
Not sure we actually got around to discussing the existing multi-touch support in the 2025-08-12 meeting.

**Comment by martinthomson** [comment]
We did, but only obliquely.  Let's touch on this at the next meeting.
